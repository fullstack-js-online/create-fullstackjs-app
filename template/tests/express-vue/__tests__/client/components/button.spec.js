import { mount } from '@vue/test-utils'
import Button from '@components/Button.vue'

describe('Button component', () => {
    it('mounts properly', () => {
        expect(
            mount(Button, {
                propsData: {
                    label: 'Register'
                }
            })
        ).toMatchSnapshot()
    })
})
