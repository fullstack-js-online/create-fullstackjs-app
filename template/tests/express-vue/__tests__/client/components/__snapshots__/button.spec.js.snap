// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component mounts properly 1`] = `<button type="button" class="focus:outline-none rounded-sm hover:bg-emerald-light px-4 py-5 w-full text-white text-xl bg-emerald"><span>Register</span></button>`;
